client/state/data-layer/wpcom/users/auth-options/index.js

```diff
@@ -72,7 +70,7 @@ export const receiveError = ( { dispatch }, action, error ) => {
 const getAuthAccountTypeRequest = dispatchRequest(
 	getAuthAccountType,
 	receiveSuccess,
-	receiveError,
+	receiveError
```

Comment:
This one is futuristic and interesting. :-) Would be pretty cool to allow these, but not in this PR.

https://eslint.org/docs/rules/comma-dangle

Reply:
so there was a period of time where i had all trailing commas allowed.
but that mean't we couldn't use it on the server because it was incompatible with our version of node.

I'm pretty sure now that we've upgraded node, we can allow for trailing commas again